### PR TITLE
wl-paste: add page, wl-copy: update link

### PR DESCRIPTION
**Pull request by ansavanix**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/wl-paste.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# wl-paste
+
+> Tool to access data stored in the clipboard for Wayland.
+> See also: `wl-copy`.
+> More information: <https://manned.org/wl-paste>.
```
Comment (marchersimon) [inline pages/linux/wl-paste.md:5]: We should use the official homepage here. This probably should also be updated for `wl-copy`.
```suggestion
> More information: <https://github.com/bugaevc/wl-clipboard>.
```

File: pages/linux/wl-paste.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- Output the contents of the clipboard:
+
+`wl-paste`
+
+- Paste the contents of the clipboard once then clear the clipboard:
```
Comment (marchersimon) [inline pages/linux/wl-paste.md:11]:
```suggestion
- Paste the contents of the clipboard and then clear it:
```

File: pages/linux/wl-paste.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+`wl-paste --paste-once`
+
+- Paste the contents of the clipboard to a file:
```
Comment (marchersimon) [inline pages/linux/wl-paste.md:15]:
```suggestion
- Write the contents of the clipboard to a file:
```

File: pages/linux/wl-paste.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# wl-paste
+
+> Tool to access data stored in the clipboard for Wayland.
+> See also: `wl-copy`.
+> More information: <https://manned.org/wl-paste>.
+
+- Output the contents of the clipboard:
```
Comment (marchersimon) [inline pages/linux/wl-paste.md:7]:
```suggestion
- Paste the contents of the clipboard:
```

**Comment by ansavanix** [comment]
Would changing the upstream link for wl-copy be in a different pull request?

**Comment by marchersimon** [comment]
I think it should be fine to include it here.

**Review by marchersimon — APPROVED** [review]
Thanks @ansavanix

**Pull request merged**